presentation/src/main/java/me/androidbox/presentation/agenda/viewmodel/AgendaViewModel.kt

```diff
@@ -62,19 +62,30 @@ class AgendaViewModel @Inject constructor(
             .minusSeconds(1L)
     }
 
-    fun fetchAgendaItems(agendaDate: ZonedDateTime = ZonedDateTime.now(ZoneId.systemDefault())) {
+    fun fetchAgendaItems(agendaDate: ZonedDateTime = ZonedDateTime.now(ZoneId.systemDefault()), isSwipeToRefresh: Boolean = false) {
```

Comment:
I added a flag here as I only want to show the swipeRefresh progress indicator only when the user has swiped to refresh. Otherwise this will show each time the user fetches new agenda items i.e. adding, changing dates, etc. Is this ok?

Reply:
Yes that works